apps/worker/src/app/workflow/usecases/message-matcher/message-matcher.usecase.ts

```diff
@@ -153,19 +159,7 @@ export class MessageMatcher {
       type = 'online';
```

Comment:
This is step specific so I keep it in the class and just pass it on tho the Filter class.